sqlalchemy_media/tests/helpers/testcases.py

```diff
@@ -20,13 +20,13 @@ def setUp(self):
         self.Base = declarative_base()
         self.engine = create_engine(self.db_uri, echo=False)
 
-    def create_all_and_get_session(self):
+    def create_all_and_get_session(self, **options):
```

Comment:
Use `expire_on_commit` as keyword argument with default value of `False` instead of `**options` and please write some docstring for method and describe the arguments.